index.js

```diff
@@ -316,6 +314,23 @@ function parseAdapterFlags(adapter) {
 const requestLink = 'https://webgpufundamentals.org/webgpu/lessons/webgpu-limits-and-features.html';
 const requestHint = 'limits greater than default must be specified when requesting adapter';
 
+function adapterWarnings(adapter, device) {
```

Comment:
Why passing `adapter` if it's not used? We can add it later when needed.

Reply:
done